### PR TITLE
Backstage interview with Yakov Pechersky

### PR DESCRIPTION
**Pull request by jcommelin**

### CODE REVIEW

**Comment by pechersky** [comment]
IMG_8319

File: posts/backstage-with-pechersky.md [hunk 1]
```diff
@@ -0,0 +1,576 @@
+---
+author: 'Johan Commelin'
+category: 'Announcements'
```
Comment (jcommelin) [inline posts/backstage-with-pechersky.md:3]:
```suggestion
category: 'Interviews'
```

File: posts/backstage-with-pechersky.md [hunk 2]
```diff
@@ -0,0 +1,576 @@
...
+I've just sort of been exploring these unfinished corners.
+
+With rational functions, I find it particularly interesting, because we have a diamond.
+From polynomials, you can go to rational functions, and then to Laurent series.
+Or you can go from polynomials to power series, and then to Laurent series.
+
+Anyway, long story short:
+I want to take rational functions, and be able to talk about roots, and poles, etc.
+And also connect this corner of algebra to analysis,
+to the work that people liky Yury Kudryashov and Patrick Massor have been doing.
```
Comment (ocfnash) [inline posts/backstage-with-pechersky.md:201]:
```suggestion
to the work that people like Yury Kudryashov and Patrick Massot have been doing.
```

Comment (PatrickMassot) [reply posts/backstage-with-pechersky.md]: Be careful that Johan fixed only on of those two typos.

**Comment by pechersky** [comment]
How about this crop? image

**Pull request merged**